custom/icds_reports/static/js/directives/sortable-kpi/sortable-kpi.directive.js

```diff
@@ -0,0 +1,27 @@
+var url = hqImport('hqwebapp/js/initial_page_data').reverse;
+
+
+function SortableKpiController($rootScope, $scope) {
```

Comment:
what is `$rootScope` used for?

Reply:
I guess your question is "what is $rootScope used for in this component?"
I copied it from some other directive, and I forgot to remove it (both $rootscope and $scope)
I will remove it now